tests/test_stress.py

```diff
@@ -4,72 +4,79 @@
 import pytest
 import random
 
-from hopr import HoprdAPI
+from .conftest import TICKET_PRICE_PER_HOP, default_nodes
+from .hopr import HoprdAPI
+from .test_integration import create_channel, passive_node, send_and_receive_packets_with_pop
 
-HTTP_STATUS_CODE_OK = 200
-HTTP_STATUS_CODE_SEND_MESSAGE_OK = 202
+
+async def check_connected_peer_count(me, count):
+    while True:
+        if len([x["peer_id"] for x in await me.peers()]) >= count:
+            break
+        else:
+            await asyncio.sleep(0.5)
 
 
 @pytest.mark.asyncio
-@pytest.mark.skipif(os.getenv("CI", "false") == "true", reason="stress tests fail randomly on CI due to resources")
+@pytest.mark.skipif(os.getenv("CI", "false") == "true", reason="stress tests fail randomly on CI due to resource constraints")
+@pytest.mark.parametrize(
+    "src,dest", [(random.choice(default_nodes()), passive_node())]
+)
+async def test_hoprd_stress_1_hop_to_self(src, dest, swarm7, cmd_line_args):
```

Comment:
Should we generalize the number of hops as a parameter?

Reply:
That is not completely generalizable. In 1-hop to self we can actually open channels to the relay and the relay will likely open a channel to us... but on a multihop... we'd have to control the entire chain. That does not translate to modifying this test to run over a staging network.